content/ios/query.md

```diff
@@ -106,13 +98,43 @@ SKYDatabase *privateDB = database;
 }];
 ```
 
-## Limiting and Offset
+## Complex Queries
+
+### Sorting the records
+We can sort records returned like this:
+
+```obj-c
+NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"_updated_at" ascending:NO];     // sorted by modificationDate
+query.sortDescriptors = @[sortDescriptor];     // apply the NSSortDescriptor to the query
+```
+
+`SKYQuery` utilizes `NSPredicate` to apply filtering on query results. You can use other parameters to sort your quries.
+
+### Limiting and Offset
 
-## Cached Queries
+We can limit the numbers of records returned like this:
 
-## Record Count
+```obj-c
+query.limit = 10;     // only show the top 10 records
+```
+
+We can also set an offset number to the query like this:
```

Comment:
use "by" instead of "like this". I think you can also mention that the behaviour is the same as SQL offset